### DISCUSSION TITLE
how can we buy you a [beverage of choice]

### DISCUSSION THREAD

**Discussion by lephtHanded** [henrygd/beszel#154] [Support] [answered]
I like to support (what I can afford :) ) projects I use when I can.... how can I buy you some coffee?

**Comment by henrygd** [comment, marked as answer]
Thanks! I'm glad you find the project useful, and I appreciate your perspective on OSS.

As of now I'm not accepting contributions. [This post by the creator of PocketBase](https://github.com/pocketbase/pocketbase/discussions/4199) mostly sums up my thoughts, however I may try GitHub Sponsors in the future if Beszel continues to grow in popularity.

That said, I will be looking for work in a few months if anyone wants to support me financially be offering me an actual job :sweat_smile:
